### PR TITLE
CHECK-77: Folders, collections and dynamic lists

### PR DESCRIPTION
**Pull request by caiosba**

### CODE REVIEW

**Comment by caiosba** [comment]
@melsawy I appreaciated your review in the PR - it's also important for you to be aware of these changes to the API and its data model.

File: app/models/project_group.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class ProjectGroup < ActiveRecord::Base
+  validates_presence_of :title, :team_id
+
```
Comment (melsawy) [inline app/models/project_group.rb:3]: @caiosba I think we should also add a validation for uniqueness title per team

**Review by melsawy — APPROVED** [review]
LGTM Dr Caio, just added a minor validation for ProjectGroup

File: app/models/project_group.rb (hunk 1 above)
Comment (caiosba) [reply app/models/project_group.rb:4]: Thanks Sawy, done!

**Pull request closed without merging**